conneg-by-ap/index.html

```diff
@@ -1240,10 +1240,10 @@ <h4>QSA Functional Profile description</h4>
         </p>
         <div class="note">
           Conformance of a system to any one, or perhaps multiple, of these profiles may be demonstrated as per
-          <a href="#conformancetoprofiles"></a>. Note that it is not required that a system, perhaps a server, indicate
-          conformance to one or more profiles of this specification since not all servers will have the ability to
-          either produce information as per <a href="#conformancetoprofiles"></a>. Systems may also show conformance to
-          the specification or profile in other ways, such as by a listing in a services catalogue. services catalogue).
+          <a href="#conformancetoprofiles"></a>. Note that it is not required that a system indicate
+          conformance to one or more profiles since not all systems will have the ability to
+          produce information as per <a href="#conformancetoprofiles"></a>. Systems may show conformance to
+          the specification or profile in other ways, such as by a listings in a services catalogue.
```

Comment:
Does _profile_ here mean _functional profile_? If so, we should spell it out.

Reply:
Sorry, PR clash. Fixed now